Cli/Formatter/MarkdownTableBuilder.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace SensioLabs\Insight\Cli\Formatter;
+
+use SensioLabs\Insight\Sdk\Model\Violation;
+use SensioLabs\Insight\Sdk\Model\Violations;
+use function sprintf;
+
+class MarkdownTableBuilder
+{
+    public static function build(Violations $violations): string
```

Comment:
did you consider using TextDescriptor or a new descriptor instead of introducing a new way to generate a specific format?